saleor/discount/migrations/0075_promotionrule_gifts.py

```diff
@@ -15,7 +15,7 @@ class Migration(migrations.Migration):
             name="gifts",
             field=models.ManyToManyField(
                 blank=True,
-                related_name="_discount_promotionrule_gifts_+",
+                related_name="+",
```

Comment:
Django 4.2 starts to distinguish between `related_name` set to `"+"` and `"_discount_promotionrule_gifts_+"`, generating a migration if `related_name` is set to `"+"` in the model (as is the case here). Changing it here prevents new, unnecessary migration (with no SQL operations) from being created.